t/plugin/basic-auth.t

```diff
@@ -189,6 +190,7 @@ Authorization: Basic Zm9vOmZvbwo=
 
 
 === TEST 8: verify
+--- yaml_config eval: $::yaml_config
```

Comment:
We don't need this line?

Reply:
Yep, will remove it. Was left over from my first attempt at changing the log level.